packages/@aws-cdk/aws-rds/test/serverless/integ.serverless-cluster.ts

```diff
@@ -0,0 +1,35 @@
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as kms from '@aws-cdk/aws-kms';
+import * as cdk from '@aws-cdk/core';
+import { DatabaseClusterEngine, ParameterGroup, ServerlessDatabaseCluster } from '../../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-rds-integ');
+
+const vpc = new ec2.Vpc(stack, 'VPC', { maxAzs: 2 });
+
+const params = new ParameterGroup(stack, 'Params', {
+  engine: DatabaseClusterEngine.AURORA_MYSQL,
+  description: 'A nice parameter group',
+  parameters: {
+    character_set_database: 'utf8mb4',
+  },
+});
+
+const kmsKey = new kms.Key(stack, 'DbSecurity');
```

Comment:
Probably unit tests are enough for `storageEncryptionKey` (makes the expected file at least a little more readable).

Reply:
done